src/review.rs

```diff
@@ -324,8 +333,8 @@ pub fn review(review_args: &ReviewArgs) -> Result<Review> {
                 }
             }
         } else {
-            // Ditto.
-            (1., Acceptance::Agree)
```

Comment:
I'm not sure why it is `Agree`

Reply:
When akochan runs in rule-based mode (high shanten), it will not calculate the expected pt.